### ISSUE TITLE
How to do a prediction in tensorflow  ? 

### ISSUE THREAD

**Issue by epratheeban** [wolfib/image-classification-CIFAR10-tf#4]
Hi,

This tutorial is amazing for a beginner like me.

I have a question about the prediction.

Let's say I have 3 section of data. (I'm not using CIFAR data)

1. Training data
2. Test data
3. Completely foreign data without any labels.

I obtained a training score and test score using your scripts. Now I want to deploy the model in the application. The tensorflow don't have a straightforward syntax for prediction. 
It would be helpful if you could add how to make a prediction after restoring the model?

Thanks 

Pratheeban
